regulations/urls.py

```diff
@@ -30,7 +31,7 @@
 newer_version_pattern = meta_version % 'newer_version'
 notice_pattern = meta_version % 'notice_id'
 
-reg_pattern = r'(?P<label_id>[\d]+)'
+reg_pattern = r'(?P<label_id>[\w]+)'
```

Comment:
Mmm, I think this might have unintended consequences. Let's keep the `reg_pattern` (which is used by other endpoints) accepting only digits and add a new pattern for document ids.